pkg/asset/images.go

```diff
@@ -12,4 +12,5 @@ var DefaultImages = ImageVersions{
 	KubeDNSMasq:     "gcr.io/google_containers/k8s-dns-dnsmasq-nanny-amd64:1.14.1",
 	KubeDNSSidecar:  "gcr.io/google_containers/k8s-dns-sidecar-amd64:1.14.1",
 	PodCheckpointer: "quay.io/coreos/pod-checkpointer:4e7a7dab10bc4d895b66c21656291c6e0b017248",
+	Fluentd:         "quay.io/abhinavdahiya/bootkube-fluentd:latest",
```

Comment:
Needs to host it under quay.io/coreos, do you have access to that? @abhinavdahiya